buildSrc/src/main/java/com/osacky/flank/gradle/validation/ValidateExclusionUsage.kt

```diff
@@ -0,0 +1,21 @@
+package com.osacky.flank.gradle.validation
+
+import com.osacky.flank.gradle.FladleConfig
+
+fun checkForExclusionUsage(config: FladleConfig) {
+  val usersProperties = config
+    .getPresentProperties()
+    .map { it.name }
+
+  exclusions.forEach {
+    if (usersProperties.contains(it.first) && usersProperties.contains(it.second)) {
+      throw IllegalStateException("Options ${it.first} and ${it.second} cannot be used both! Choose one of them.")
+    }
+  }
+}
+
+private val exclusions = listOf(
+  "testShards" to "maxTestShards",
+  "testShards" to "numUniformShards",
```

Comment:
oh, i didn't know that this couldn't be used together. we should also specify this in the docuementation for maxTestShards that this cannot be used with this property.